tensorflow/training/docker/2.4/py3/cu110/Dockerfile.gpu

```diff
@@ -115,10 +115,11 @@ RUN wget https://sourceforge.net/projects/boost/files/boost/1.73.0/boost_1_73_0.
  && tar -xzf boost_1_73_0.tar.gz \
  && cd boost_1_73_0 \
  && ./bootstrap.sh \
- && ./b2 threading=multi --prefix=/usr -j 64 cxxflags=-fPIC cflags=-fPIC install \
+ && ./b2 threading=multi --prefix=/usr -j 64 cxxflags=-fPIC cflags=-fPIC install || true \
  && cd .. \
  && rm -rf boost_1_73_0.tar.gz \
- && rm -rf boost_1_73_0
+ && rm -rf boost_1_73_0 \
+ && cd /usr/include/boost
```

Comment:
Is this a validation step to see if the build succeeded?

Reply:
to see if boost installation succeeded